vendor/assets/iD/iD.css.erb

```diff
@@ -788,29 +807,41 @@ path.stroke.tag-natural-wetland {
 
 /* Light Green things */
 path.stroke.tag-landuse-cemetery,
-path.stroke.tag-landuse-orchard,
-path.stroke.tag-landuse-vineyard,
+path.stroke.tag-landuse-farmland,
+path.stroke.tag-landuse-grass.tag-golf-green,
 path.stroke.tag-landuse-meadow,
-path.stroke.tag-landuse-farmland {
+path.stroke.tag-landuse-orchard,
+path.stroke.tag-landuse-vineyard {
     stroke: rgb(191, 232, 63);
 }
+path.fill.tag-landuse-cemetery,
+path.fill.tag-landuse-farmland,
+path.fill.tag-landuse-grass.tag-golf-green,
+path.fill.tag-landuse-meadow,
+path.fill.tag-landuse-orchard,
+path.fill.tag-landuse-vineyard {
+    stroke: rgb(191, 232, 63, 0.3);
+    fill: rgb(191, 232, 63, 0.3);
```

Comment:
Those two lines also need to be changed from rgb to rgba

Reply:
Hmm. this is already fixed in https://github.com/openstreetmap/iD/commit/acfd1af5b236a876bb9eb4e9b1fd6e402645c347, but I don't see, to find the fix for natural-wetland upstream (https://github.com/openstreetmap/iD/blob/master/css/25_areas.css#L197-L198). Somehow those two fixes are now a bit mixed up between repos. 🤷‍♂️

Reply:
@mmd-osm Okay thanks, I just fixed this on that branch too.

Reply:
Ok thanks! Will you create another PR for this repo that fixes vineyards?

Reply:
Ohhhh I wasn't reading carefully and didn't realize there were two separate issues here 😂

Will fix now.